utils/ratlib.py

```diff
@@ -188,6 +188,39 @@ def duration(time: datetime.timedelta) -> str:
     return humanfriendly.format_timespan(time, detailed=False, max_units=2)
 
 
+def correct_system_name(system: str) -> str:
```

Comment:
Can we stop piling unrelated functions in `ratlib.py`? If @theunkn0wn1 agrees, I'd like you to move this into its own module under `utils/`.

Reply:
I would argue `ratlib.py` is horribly named if this function is unrelated.

According to the module docstring its for
```
ratlib.py - IRC/Fuelrat nickname utilities
```
yet its named as `ratlib`. 

Therefore, the docstring and module's name don't match, which leads to the confusion on what the file is actually for.
`ratlib` suggests is a generic library`, the docstring suggests its specifically for nickname utilities.

Reply:
Regardless of the docstring, I see the problem in collecting unrelated functions and classes in a single module. It's not a crime to have modules containing only a single function or class, you know.

Reply:
Resolved via its own file.